src/main/java/com/formstack/qa/pages/BillingMatrixPage.java

```diff
@@ -42,7 +42,7 @@
     @FindBy(xpath = "//a[contains(@href,'bronze')][2]")
     private WebElement switchToBronzeTrial;
 
-    @FindBy(xpath = "//p[text()=\"Deactivate Account\"]")
+    @FindBy(xpath = "//span[@data-test=\"deactivate_account_matrix_page\"]//p[@class=\"fs-btn2__text\"]")
```

Comment:
After discussion - let's go with 
```suggestion
    @FindBy(xpath = "//span[@data-test=\"deactivate_account_matrix_page\"]//p[text()=\"Deactivate Account\"]
```
as it should be easier to debug rather than magical class.